sources/advanced/Cli.ts

```diff
@@ -16,12 +16,25 @@ const errorCommandSymbol = Symbol(`clipanion/errorCommand`);
 type MakeOptional<T, Keys extends keyof T> = Omit<T, Keys> & Partial<Pick<T, Keys>>;
 type VoidIfEmpty<T> = keyof T extends never ? void : never;
 
+let tty: typeof ttyType | undefined;
+try {
+  tty = require(`tty`) as typeof ttyType;
+} catch {}
```

Comment:
Not related to this diff, I just wanted to silence the build errors when building Clipanion w/ Webpack for browser environments.